src/kernels/raw/finder/localKernelSpecFinderBase.node.ts

```diff
@@ -254,8 +254,7 @@ export async function loadKernelSpec(
     let kernelJson: ReadWrite<IJupyterKernelSpec>;
     try {
         traceVerbose(
-            `Loading kernelspec from ${getDisplayPath(specPath)} for ${
-                interpreter?.uri ? getDisplayPath(interpreter.uri) : ''
+            `Loading kernelspec from ${getDisplayPath(specPath)} for ${interpreter?.uri ? getDisplayPath(interpreter.uri) : ''
```

Comment:
Yeah I'm guessing you don't have prettier formatting on save (or it's formatting differently). The linter step in the build is likely going to barf on this one.

Reply:
Good catch, I have a setting which enforces ts formatter thus it behaves differently.